src/lib.rs

```diff
@@ -117,13 +117,8 @@
 //! // conventions.
 //! #[allow(non_snake_case)]
 //! pub extern "system" fn Java_HelloWorld_hello(env: JNIEnv,
-//! // this is the class that owns
-//! our
-//! // static method. Not going to
-//! be
-//! // used, but still needs to
-//! have an
-//!                                              // argument slot
+//! // this is the class that owns our static method. Not going to be used, but
+//! // still needs to have an argument slot.
```

Comment:
I'd explain _why_ it must be here:
`but still must be present to match an expected signature of a static native method.`